spec.md

```diff
@@ -23,6 +23,7 @@
 - [API](#api)
 	- [Endpoints](#endpoints)
 	- [Error Codes](#error-codes)
+    - [Warnings](#warnings)
```

Comment:
still seems off compared to Error Codes?

Reply:
Did we mix tabs and spaces in the TOC?